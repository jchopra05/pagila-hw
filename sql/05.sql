/*
 * Count the total number of G rated films.
 * HINT:
 * Use the count() aggregation function and the rating column of the film table.
 */

SELECT COUNT(*) AS total_g_rated_films
FROM public.film
WHERE rating = 'G';
