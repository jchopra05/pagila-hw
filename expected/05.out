 total_g_rated_films 
---------------------
                 178
(1 row)

